### PR TITLE
improve prod deployment docs wrt ansible inventories

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: docs/production.md [hunk 1]
```diff
@@ -89,8 +90,10 @@ After installing a Katello server, you could then spin up a Capsule with the ass
 
 For a release version in production:
 
-    ansible-playbook -l localhost playbooks/katello.yml -e foreman_repositories_version=1.14 -e katello_repositories_version=3.3
+    ansible-playbook --connection=local --inventory=localhost, -l localhost playbooks/katello.yml -e foreman_repositories_version=1.14 -e katello_repositories_version=3.3
 
 For nightly production:
 
-    ansible-playbook -l localhost playbooks/katello.yml
+    ansible-playbook --connection=local --inventory=localhost, -l localhost playbooks/katello.yml
```
Comment (ehelms) [inline docs/production.md]: I'd like to simplify this to having an inventory in `inventories/` that looks like:

```
localhost ansible_connection=local
```

File: docs/production.md [hunk 2]
```diff
@@ -66,6 +66,7 @@ This use case assumes that you are on a box that has SSH access to the target bo
 1. Ensure Ansible is installed
 2. Clone this repository - `git clone https://github.com/theforeman/forklift.git`
 3. Enter the repository - `cd forklift`
+4. [Create an inventory file](https://docs.ansible.com/ansible/latest/intro_inventory.html) with your hosts in the `inventories` subfolder. You can also pass a different inventory to `ansible-playbook` using `--inventory`.
```
Comment (ehelms) [inline docs/production.md]: using `--inventory` or `-i`.

File: docs/production.md (hunk 1 above)
Comment (evgeni) [reply docs/production.md]: we can do that, yeah.

**Comment by evgeni** [comment]
@ehelms updated

**Comment by alvinstarr** [comment]
I have run a test install using the instructions and they work.

**Pull request merged**